free-programming-books.md

```diff
@@ -1328,6 +1331,11 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Using C on the UNIX System](http://www.bitsinthewind.com/about-dac/publications/using-c-on-the-unix-system) - David A. Curry
 
 
+### Front-End
+
+* [Atomic Design](https://atomicdesign.bradfrost.com) - Brad Frost
```

Comment:
It's not helpful to add a category for a single resource. Try to find another category.